18/task3.py

```diff
@@ -0,0 +1,17 @@
+"""
+HomeWork 14 - task 3
+Dmytro Verovkin
+robot_dreams
+
+3.(необов'язкове виконання) Створити клас MyStr(str), який має перевизначтити метод str таким чином,
+щоб замість друку реального значення всі літери були переведені в верхній регістр:
+"""
+
+class MyStr(str):
+    def __str__(self):
+        return super().__str__().upper()
```

Comment:
Also, you can do it like this:
`self.upper()` 
Since `self` is a string and can be transformed to uppercase using `.upper`